platforms/python/versions/bullseye/versionsToBuild.txt

```diff
@@ -1,24 +1,25 @@
+
+# NOTE: Make sure to set the default version in 'defaultVersion.txt' file
 # version, gpg keys, dockerfile
-3.7.9, 0D96DF4D4110E5C43FBFB17F2D347EA6AA65421D,
+3.10.0, A035C8C19219BA821ECEA86B64E628F8D684696D,
```

Comment:
Was there a change in our version sorting logic that's causing it to be sorted by lexicographical value rather than version value?

Reply:
That's something we've done [for a while](https://github.com/microsoft/Oryx/blob/71ed1de2a364b8d561fd6e24252749cb14148c06/build/tools/Automation/Services/FileService.cs#L46).